pkgs/development/python-modules/block-io/default.nix

```diff
@@ -1,8 +1,9 @@
-{ stdenv, fetchPypi, buildPythonPackage, base58, ecdsa, pycryptodome, requests, six, setuptools }:
+{ stdenv, fetchPypi, buildPythonPackage, base58, ecdsa, pycryptodome, requests, six, setuptools, isPy27 }:
 
 buildPythonPackage rec {
   pname = "block-io";
   version = "1.1.15";
+  disabled = isPy27; # base58 dependency dropped python 2 support
```

Comment:
please add this `disabled` clause to the base58 package, instead of this package please

Reply:
good catch, it was already disabled on base58. I thought we need to be explicit on python 2 support here too

Reply:
no, an error will be thrown on `base58` when evaluating, thus making this fail evaluation.